tests/check_sigstore_signer.py

```diff
@@ -8,20 +8,13 @@
 import os
 import unittest
 
-from sigstore.oidc import detect_credential  # pylint: disable=import-error
-
 from securesystemslib.signer import (
-    KEY_FOR_TYPE_AND_SCHEME,
-    Key,
-    SigstoreKey,
+    SIGNER_FOR_URI_SCHEME,
+    Signer,
     SigstoreSigner,
 )
 
-KEY_FOR_TYPE_AND_SCHEME.update(
-    {
-        ("sigstore-oidc", "Fulcio"): SigstoreKey,
-    }
-)
+SIGNER_FOR_URI_SCHEME.update({SigstoreSigner.SCHEME: SigstoreSigner})
```

Comment:
very minor nit: this might be a bit simpler for updating a single key
```suggestion
SIGNER_FOR_URI_SCHEME[SigstoreSigner.SCHEME] = SigstoreSigner
```